src/language-js/utils/index.js

```diff
@@ -475,6 +459,36 @@ function isUnitTestSetUp(node) {
   );
 }
 
+const patterns = [
```

Comment:
`patterns` is too general. What do you think about `testCallCalleePatterns` ?

Reply:
I agree.
I will use `testCallCalleePatterns`.
Thank you.